scripts/ci/check_compliance.py

```diff
@@ -264,8 +264,12 @@ def run(self):
                 line = line.strip()
                 if not line or line.startswith("#"):
                     continue
-                vendor, _ = line.split("\t", 2)
-                vendor_prefixes.append(vendor)
+                try:
+                    vendor, _ = line.split("\t", 2)
+                    vendor_prefixes.append(vendor)
+                except ValueError:
+                    self.error(f"""Invalid line in vendor-prefixes.txt:
+                    \"{line}\". Did you forget the tab character?""")
```

Comment:
this is going to print the leading space on the second line

```
ERROR   : Test BoardYml failed: 
Invalid line in vendor-prefixes.txt:
                    "adsfas". Did you forget the tab character?

Complete results in compliance.xml
```

use two separate strings

Reply:
Done